projects/forms/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 -   missing support for `ValidatorFn[]` and `AsyncValidatorFn[]` for `typedFormControl`
+-   missing support for `getRawValue()` on `TypedFormGroup`
```

Comment:
Version 1.1.2 has already been released. I guess the [unreleased] above it is a bit misleading?

The following is what denotes the next version 1.1.3 provided there are only `chore:...` and `fix:...` commit messages:

```md
## [Unreleased]

## [1.1.3] - 2021-07-09

### Fixed

-   missing support for `getRawValue()` on `TypedFormGroup`
-   incorrect install instructions in README "Getting Started" section
```